src/Owin.Security.OpenIdConnect.Extensions/OpenIdConnectConstants.cs

```diff
@@ -110,6 +124,8 @@ public static class Metadata {
             public const string SubjectTypesSupported = "subject_types_supported";
             public const string TokenEndpoint = "token_endpoint";
             public const string UserinfoEndpoint = "userinfo_endpoint";
+            //Added by Introspection (but not in Spec?)
+            public const string IntrospectionEndpoint = "introspection_endpoint";
```

Comment:
Ah, right. I forgot to do that with the configuration endpoint. I'll take this out of the current PR for now and do another PR after I update the configuration endpoint.

Also: about the styling, blame ReSharper :-1: aside from the braces, what style changes should I attempt to fix? :laughing: